pkg/k8s/rule_translate.go

```diff
@@ -221,7 +221,7 @@ func (k RuleTranslator) deleteToCidrFromEndpoint(
 	return toReleasePrefixes, nil
 }
 
-// PreprocessRules translates rules that apply to headless services
+// PreprocessRules translates rules that apply to services without selectors
```

Comment:
Thanks! I'm not familiar with the terminology here, so I may be wrong. But looking at the code and history, I see that this function would initially apply rules on headless services indeed. It changed with commit a270d52ef158371fa2be06db53df39866af0080c. Based on the code, should the comment be:

```suggestion
// PreprocessRules translates egress rules that apply to external services (ToServices)
```

Looking at the description for `ToServices`, the fact that these K8s services have no selectors seems more like the current support status and could possibly change one day?

Since 09b3df13a5f3e7232075bff6b33fd3b885726301:

```
	// ToServices is a list of services to which the endpoint subject
	// to the rule is allowed to initiate connections.
	// Currently Cilium only supports toServices for K8s services without
	// selectors.
```

Reply:
> Thanks! I'm not familiar with the terminology here, so I may be wrong. But looking at the code and history, I see that this function would initially apply rules on headless services indeed. It changed with commit https://github.com/cilium/cilium/commit/a270d52ef158371fa2be06db53df39866af0080c. Based on the code, should the comment be:

Sweet. I guess the egress part is implicit, since there is no way to have ingress rules service. I'll update the comment based on your suggestion, thanks!